documentation/source/user_section/tutorials/2/2-5.rst

```diff
@@ -0,0 +1,30 @@
+.. _choosing-labels:
+
+2.5 Extracting specific labels
+##############################
+
+You don't need to label every vertebrae in order to perform registration; you can instead choose to work with a subset of labels:
+
+* **3+ labels**: This is the most accurate method for matching the vertebral levels of the anatomical image to the levels of the template. However, there is a key downside to this approach: Because more than two points must be matched, the level-matching transformation cannot be affine. As a result, the output warping field will be undefined for regions above the top label and below the bottom label.
+* **2 labels:** In practice, the difference in accuracy between using 3+ labels and 2 labels is often negligible. Using 2 labels also has the added benefit of allowing for an affine level-matching transformation, which means the template-to-image warping field will be defined for the entire image. For these reasons, we strongly recommend starting with 2 labels for your registration.
+* **1 label:** If your image covers only 1 vertebrae, you can still provide a single label. Note that the transformation in this case will be limited to a Z-axis translation, as an affine transformation can't be determined for a single point.
+
+As starting with 2 labels is recommended, you will need to extract them from the labels that were automatically generated in the previous step. To discard the extra vertebral levels, we use ``sct_label_utils`` to create a new label image containing only 2 of the labels. These points are used to match the levels of the subject to the levels of the template, and correspond to the top and bottom vertebrae we wish to use for image registration.
+
+.. code:: sh
+
+   sct_label_utils -i t2_seg_labeled.nii.gz -vert-body 3,9 -o t2_labels_vert.nii.gz
+
+:Input arguments:
+   - ``-i`` : Input image containing a spinal cord labeled with vertebral levels
+   - ``-vert-body`` : The top and bottom vertebral levels to create new point labels for. Choose labels based on your region of interest. Here, we have chosen ``3,9`` (C3 to T1).
+   - ``-o`` : Output filename
+
+:Output files/folders:
+   - ``t2_labels_vert.nii.gz`` : Image containing the 2 single-voxel vertebral labels
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/registration_to_template/io-sct_label_utils.png
```

Comment:
The labels are there, but because they are single-voxel labels, they're hard to see with the grayscale colormap + default zoom.

![Screenshot from 2021-06-07 16-19-39](https://user-images.githubusercontent.com/16181459/121365330-8d260d00-c906-11eb-9d36-2973eed7d7d8.png)

I can definitely add a note about this to avoid any confusion. :slightly_smiling_face:

Reply:
> I can definitely add a note about this to avoid any confusion

People rarely read notes. We should instead load a colored colormap as we do at the end of sct_label_vertebrae (we can pick the same colormap)

Reply:
> We should instead load a colored colormap as we do at the end of sct_label_vertebrae (we can pick the same colormap)

:+1: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3416